### ISSUE TITLE
Tool fails in M1 intel machines

### ISSUE THREAD

**Issue by dcacenabes** [DataDog/datadog-ci#514]
**Describe what happened**
Running the command in an M1 mac machine throws the error:
```
dyld[21938]: missing symbol called
```

**Steps to reproduce the issue:**
In an M1 machine:
- install the package (either yarn or npm, does not matter) `npm install -g @datadog/datadog-ci`
- call the tool: `datadog-ci version`

**Expected behaviour:**
Command outputs v1.2.0

**Actual behaviour:**
```
dyld[22320]: missing symbol called
[1]    22320 abort      datadog-ci version
```

**Additional context**
 - Monterrey 12.3 in M1
 - Node version tried with 16.13.0 and 17.8.0
 
 I tried doing `npx @datadog/datadog-ci@1.1.0` version and it worked
If I do `npx @datadog/datadog-ci@1.2.0 version` it fails
Installing `npm install -g @datadog/datadog-ci@1.1.0` gives the error
Installing `npm install -g @datadog/datadog-ci@1.2.0` also gives the error

**Comment by m-rousse** [comment]
Hi @dcacenabes,
Thanks for raising this issue.
This is an issue with the `ssh2` and `cpu-features` libraries we rely on, it is tracked in the following issue: https://github.com/mscdex/cpu-features/issues/7
TL;DR is that the maintainer of these packages is waiting on the support for M1s in Google’s `cpu_feature` tool.

Alternatively, you should be able to install the package by discarding the optional dependencies using `--no-optional`.

**Comment by dcacenabes** [comment]
Hi @m-rousse , thanks for your quick reply!
Apparently --no-optional is not working when you install a package globally. (`npm i -g --no-optional @datadog/datadog-ci`)
We moved to installing it locally enforcing no optional parameter and launch with npx, then it worked!
Thanks for your support.
